### PR TITLE
Add Test Kitchen

### PR DESCRIPTION
**Pull request by jtimberman**
Logo from https://kitchen.ci/

### CODE REVIEW

File: img-buildkite-64.json [hunk 1]
```diff
@@ -1,4 +1,13 @@
 [
+  {
+    "name": "testkitchen",
+    "image": "img-buildkite-64/testkitchen.png",
+    "category": "Buildkite",
+    "aliases": [
+      "test-kitchen",
+      "kitchen-ci"
+    ]
```
Comment (ticky) [inline img-buildkite-64.json]: I’d make the main “name” `test-kitchen`, rather than taking two very-similar names :)

File: README.md [hunk 2]
```diff
@@ -26,6 +26,7 @@ If we're missing a Unicode emoji, please check if it's been added to our upstrea
 
 Emoji | Aliases
 ----- | -------
+<img src="img-buildkite-64/testkitchen.png" width="20" height="20" alt="testkitchen"/> | `:test-kitchen:`, `:kitchen-ci:`
```
Comment (ticky) [inline README.md]: The “aliases” column should contain all the names you can refer to this by

**Comment by jtimberman** [comment]
Heyo! This response got lost in my GitHub notifications. I'll get to the change requests later this week or early next. Thank yoU!

**Comment by toolmantim** [comment]
I’ve made those changes @ticky

**Comment by jtimberman** [comment]
Thank you ❤️ ❤️ ❤️. I took on a new role and hadn't gotten a chance to get back to this. 🎈

**Pull request merged**